CHANGELOG.md

```diff
@@ -2,24 +2,35 @@
 
 ## [1.0.0](https://github.com/ChainSafe/web3.js-plugin-chainlink/compare/v1.0.0...v1.0.0) (2023-08-07)
 
-
 ### ⚠ BREAKING CHANGES
 
-* rename npm project
+-   rename npm project
 
 ### Features
 
-* rename, update dependencies and configs. ([d29710b](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/d29710b5722de44d50cdd425808ac0631a355cfa))
-
+-   rename, update dependencies and configs. ([d29710b](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/d29710b5722de44d50cdd425808ac0631a355cfa))
 
 ### Bug Fixes
 
-* ci ([0c50eea](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/0c50eea7249adfcd3c4a81a80291aa1a71937515))
-* update blackbox tests ([ae4a18b](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/ae4a18b3f8c0f9297d90b6bcb26207f57b69dca6))
+-   ci ([0c50eea](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/0c50eea7249adfcd3c4a81a80291aa1a71937515))
+-   update blackbox tests ([ae4a18b](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/ae4a18b3f8c0f9297d90b6bcb26207f57b69dca6))
+
+### Miscellaneous
+
+-   release 1.0.0 ([5848325](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/5848325c7763507620d0beb3ce86cc235a9694bd))
+-   release 1.0.0 ([c1e0378](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/c1e03787ff79ed2d7522cddb7ff3b99391ea3893))
+-   rename npm project ([312f75d](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/312f75d64e16faead85ac5a990678b8aefae19a9))
 
+## [Unreleased]
 
 ### Miscellaneous
 
-* release 1.0.0 ([5848325](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/5848325c7763507620d0beb3ce86cc235a9694bd))
-* release 1.0.0 ([c1e0378](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/c1e03787ff79ed2d7522cddb7ff3b99391ea3893))
-* rename npm project ([312f75d](https://github.com/ChainSafe/web3.js-plugin-chainlink/commit/312f75d64e16faead85ac5a990678b8aefae19a9))
+-   Corrected the `yarn add` statement for the Chainlink web3.js plugin.
+    -   Previously, the documentation incorrectly stated:
+        ```
+        yarn add @chainsafe/web3.js-chainlink-plugin
+        ```
+        However, the correct yarn statement is:
+        ```
+        yarn add "@chainsafe/web3-plugin-chainlink";
+        ```
```

Comment:
To keep the CHANGELOG.md simple, we put one line for the change and we include the PR number. And any description we would like to associate would be at the PR description (you can edit it).
```suggestion
-   Corrected the `yarn add` statement according to the new package name (#34)
```